specifications/catalog/catalog.protocol.md

```diff
@@ -1,15 +1,13 @@
 # Catalog Protocol {#catalog-protocol}
 
-This document outlines the [=Catalog Protocol=]. The used terms are described in section [[[#terminology]]].
-
 ## Introduction
 
 The [=Catalog Protocol=] defines how a [=Catalog=] is requested from a [=Catalog Service=] by a [=Consumer=] using an
-abstract [=Message=] exchange format. The concrete [=Message=] exchange wire format is defined in the binding specifications.
+abstract [=Message=] exchange format. The concrete [=Message=] exchange wire format is defined in the binding.
 
 The [=Catalog Protocol=] reuses properties from the DCAT and ODRL vocabularies with restrictions defined in this
-specification. This is done implicitly by the use of the JSON schemas and JSON-LD-contexts that are part of the DSP.
-Servers have no obligation to process properties that are not part of the schemas.
+specification. This is done implicitly by the use of the JSON schemas and JSON-LD-contexts that are part of the [=Dataspace Protocol=].
+Servers _MUST NOT_ process properties that are not part of the schemas.
```

Comment:
This is a semantic change. `MUST NOT` is a prohibition see [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119#section-2)

Reply:
You're right. Thx! Will change to `_MAY_`.